src/qibocal/auto/runcard.py

```diff
@@ -47,21 +44,9 @@ class Runcard:
     """Structure of an execution runcard."""
 
     actions: list[Action]
-    qubits: Union[list[QubitId], list[tuple[QubitId, QubitId]]]
+    qubits: Union[list[QubitId], list[tuple[QubitId, QubitId]]] = None
```

Comment:
Strictly speaking here we need
```suggestion
    qubits: Optional[Union[list[QubitId], list[tuple[QubitId, QubitId]]]] = None
```
since it can be `None`.